src/Events/Request/Controller.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Controller for request-level sanitization in TEC.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Request
+ */
+
+namespace TEC\Events\Request;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+use TEC\Events\Request\Ical;
+
+/**
+ * Class Controller
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Request
+ */
+class Controller extends Controller_Contract {
+
+	/**
+	 * The action registration action for the events request controller.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static string $registration_action = 'tec_events_request_controller_registered';
+	/**
```

Comment:
```suggestion
	/**
	 * The action registration action for the events request controller.
	 *
	 * @since TBD
	 *
	 * @var string
	 */
	public static string $registration_action = 'tec_events_request_controller_registered';

	/**
```